flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/functions/casting/CastRule.java

```diff
@@ -88,7 +91,16 @@ public ZoneId getSessionZoneId() {
                 public ClassLoader getClassLoader() {
                     return classLoader;
                 }
+
+                @Override
+                @Nullable
+                public CodeGeneratorContext getCodeGeneratorContext() {
+                    return null;
```

Comment:
Can add some annotation to explain why here returns null?

Reply:
Change it to non null